src/components/Dashboard.vue

```diff
@@ -58,6 +58,27 @@
         </InfoButton>
       </div>
 
+      <div class="flex ml-3 md:ml-0 justify-start items-center mt-2 space-x-4">
+        <p class="text-sm w-fit">Number of days off taken in a year</p>
+        <!-- This input field accepts negative values, with a minimum limit that ensures the total number of working days in a year does not exceed 365.  -->
+        <AdjustCounter
+          :value="store.nrDaysOff"
+          @update:value="store.setNrDaysOff"
+          :min="YEAR_BUSINESS_DAYS - 365" 
```

Comment:
I don't agree with this `min` value. It's not clear that a negative number means "I work during the weekends" (or something like that). I rather leave it to 0 and make the `YEAR_BUSINESS_DAYS` variable editable somewhere (not necessarily in this PR, we can do it after)

Reply:
Agree. 
I wanted to give the user the option to increase the number of working days in a year to a maximum of 365 days. 
On a second though, I agree that this should not be implemented this way.

Resolved: set min to 0